plugins/flutter_plugins/url_launcher_fde/ios/Classes/UrlLauncherFakeFdePlugin.m

```diff
@@ -0,0 +1,6 @@
+#import "UrlLauncherFakeFdePlugin.h"
+
+@implementation UrlLauncherFakeFdePlugin
```

Comment:
FDEURLLauncherPlugin

Reply:
completely removed the class, the podspec seems to be enough..